tunelinux/iscsi.go

```diff
@@ -364,6 +364,11 @@ func SetIscsiRecommendations() (err error) {
 					continue
 				}
 			}
+			if recommendation.Parameter == "replacement_timeout" {
```

Comment:
@skshetti i thought we will introduce --global parameter to configure settings at host level or bypass otherwise. Are you not taking that approach?